tic-tac-toe.js

```diff
@@ -1,7 +1,91 @@
 function TicTacToe() {
-
+  this.boardTiles = [["row-0-col-0", "row-0-col-1", "row-0-col-2"],
+    ["row-1-col-0", "row-1-col-1", "row-1-col-2"],
+    ["row-2-col-0", "row-2-col-1", "row-2-col-2"]];
+  this.turn = "X";
 }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.play = function() {
+  var self = this;
+
+  self.boardTiles.forEach( function(row, row_index) {
+    row.forEach(function(tile, tile_index) {
+      $('#' + tile).one("click", function() {
+        // add a p tag with X or O to correct div for front end
+        $('#' + tile).append("<p class='clicked'>" + self.turn + "</p>");
+        // replace boardTiles string with X or O for backend
+        self.boardTiles[row_index][tile_index] = self.turn;
+        // check to see if the game is won
+        if (self.isGameWon()) {
+          // announce winner
+          $('#tic-tac-toe').append("<h3>Player " + self.turn + " won!</h3>");
+          // prevent divs from being clicked again
+          self.clicksOff(self);
+          // display button to play again
+          $('#tic-tac-toe').append("<form><input type='button' onClick='history.go(0)' value='Play Again'></form>");
+        } else if (self.isGameTied()) {
+          // announce tie
+          $('#tic-tac-toe').append("<h3>Tie!</h3>");
+          // turn clicks off
+          self.clicksOff(self);
+          // display button to play again
+          $('#tic-tac-toe').append("<form><input type='button' onClick='history.go(0)' value='Play Again'></form>");
```

Comment:
There's a bit of repetition here `if isGameTied` to `if isGameWon`. Could possibly have taken this repetition out, but that's me being nitpicky. :)